Wikipedia/Code/WebViewController.m

```diff
@@ -447,17 +581,15 @@ - (void)scrollToFragment:(NSString*)fragment animated:(BOOL)animated {
         [self.webView.scrollView scrollRectToVisible:CGRectMake(0, 1, 1, 1) animated:animated];
     } else {
         if (!animated) {
-            [self.webView.wmf_javascriptContext.globalObject invokeMethod:@"scrollToFragment"
-                                                            withArguments:@[fragment]];
+            [self.webView evaluateJavaScript:[NSString stringWithFormat:@"window.wmf.utilities.scrollToFragment('%@')", fragment] completionHandler:nil];
```

Comment:
A good idea is maybe to put all code that evaluates javascript into a category on WKWebView. (this goes for the the previous comment as well)

That way we can isolate js stuff from the webviewcontroller.